runtimes/bulletin-westend/tests/tests.rs

```diff
@@ -178,6 +201,66 @@ fn transaction_storage_runtime_sizes() {
 	});
 }
 
+/// Test maximum write throughput: 8 transactions of 1 MiB each in a single block (8 MiB total).
+#[test]
+fn transaction_storage_max_throughput() {
+	use bulletin_westend_runtime as runtime;
+	use bulletin_westend_runtime::BuildStorage;
+	use frame_support::assert_ok;
+	use pallet_transaction_storage::{AuthorizationExtent, Call as TxStorageCall};
+	use sp_keyring::Sr25519Keyring;
+
+	const NUM_TRANSACTIONS: u32 = 8;
+	const TRANSACTION_SIZE: usize = 1 * 1024 * 1024; // 1 MiB
```

Comment:
clippy? https://github.com/paritytech/polkadot-bulletin-chain/actions/runs/20304830868/job/58320702921?pr=148
```suggestion
	const TRANSACTION_SIZE: usize = 1024 * 1024; // 1 MiB
```